api/v1beta1/hostedcluster_types.go

```diff
@@ -626,6 +626,40 @@ type PlatformSpec struct {
 	// +optional
 	// +immutable
 	PowerVS *PowerVSPlatformSpec `json:"powervs,omitempty"`
+
+	// KubeVirt defines KubeVirt specific settings for cluster components.
+	//
+	// +optional
+	// +immutable
+	Kubevirt *KubevirtPlatformSpec `json:"kubevirt,omitempty"`
```

Comment:
isn't the correct notation should be `KubeVirt`?

Reply:
for consistency I am using the same notation that we use in the NodePool api though. example...

```
        // Kubevirt specifies the configuration used when operating on KubeVirt platform.
        //
        // +optional
        Kubevirt *KubevirtNodePoolPlatform `json:"kubevirt,omitempty"`
```

I wouldn't be opposed to transitioning to use `KubeVirt` across the board within hypershift in the future. This president of using `Kubevirt` unfortunately got set within capk as part of Apple donating that code. It has carried its way into hypershift due to the KubevirtMachine objects and other apis like that. That's a larger change than i'd like to make right now though. So i'm favoring consistency instead